### DISCUSSION TITLE
testing print_unicode

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#61] [Q&A] [answered]
So how can I make a performance rating task for print_unicode?

**Comment by VendelinSlezak** [comment, marked as answer]
Hello, I apologize for this few day pause, but now I am back, ready to continue improving BleskOS :) You can use \x to add byte hex value. For example unicode value for A is 0x0041, so you will print it like "\x41\x00". And on end there need to be char 0x0000, so full command to print "A" in unicode would look like this: `print_unicode("\x41\x00\x00\x00", 10, 10, BLACK);` But I do not think testing print_unicode is really needed, because it is exactly same method as print_ascii, only difference is that first take always word value from string and second byte value, so it should not make performance difference. By the way, I like how you improved performance editor :)
